docs/run/nodes/nodes.md

```diff
@@ -177,7 +175,7 @@ ceramic daemon
         ]
     },
     "ipfs": {
-        "mode": "remote", // Use "remote" or "bundled"
+        "mode": "remote", // Use "remote" or "bundled". "remote" is recommended for production deployments
```

Comment:
Maybe just remove this comment altogether, re: comments above

Reply:
done